<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>952</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QTCPServer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(140, 140, 140);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QPushButton" name="pushButton_sendMessage">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(31, 178, 255);
color:white;
border-width: 2px;
border-style: solid;
border-color: rgb(43, 162, 255);
border-radius: 10px;
padding: 5px;
</string>
      </property>
      <property name="text">
       <string>Envoyer</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QComboBox" name="comboBox_receiver">
      <property name="minimumSize">
       <size>
        <width>110</width>
        <height>0</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>Broadcast</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLineEdit" name="lineEdit_message"/>
    </item>
    <item row="5" column="0">
     <widget class="QTextBrowser" name="textBrowser_receivedMessages">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0" alignment="Qt::AlignHCenter">
       <widget class="QLineEdit" name="lineEdit_lattitude">
        <property name="text">
         <string>3000</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Lattitude</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" alignment="Qt::AlignHCenter">
       <widget class="QLineEdit" name="lineEdit_longitude">
        <property name="text">
         <string>4000</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Longitude</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="Digital_Clock">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>CLOCK</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>952</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
